camera_calib_path: # in degree [base, shoulder, elbow, wrist1, wrist2, wrist3]
  pos_0: [-180.3, -121.3, -112.2, -40.2, 90.5, -45.0]
  pos_1: [-213.9, -107.4,  -70.6, -89.7, 94.6, -52.5]
  pos_2: [-180.3, -138.7,  -21.3,-127.2, 90.5, -44.4]
  pos_3: [-177.6, -107.0,  -71.8, -86.7, 94.8, -52.5]
  pos_4: [-180.3, -120.9,  -54.6, -98.3, 90.5, -44.6]
  pos_5: [-156.8, -114.1,  -39.8,-118.5, 95.2, -52.4]
  pos_6: [-180.3,  -98.4,  -85.2, -90.2, 90.4, -44.7]
  pos_7: [-150.6,  -93.2,  -66.6,-111.9, 95.4,   0.0]
  pos_8: [-180.3, -108.7, -107.2, -57.9, 90.4, -44.9]
  pos_9: [-145.9,  -93.9,  -96.4,-101.4, 67.8,  10.2]
  pos_10: [-199.4,  -61.3,-119.2, -64.0, 95.3,  28.5]
  pos_11: [-154.5, -103.0, -94.1, -90.2, 69.6, 172.9]
  pos_12: [-209.8, -140.5, -79.3, -77.5,106.7, 109.5]
  pos_13: [-193.8, -115.8, -23.0,-136.5, 92.5, 114.3]
  pos_14: [-189.3, -149.8, -37.4,-108.4, 89.6, 310.1]
  pos_15: [-137.4, -170.4, -27.4,-117.8, 72.6,  10.0]

backup_pos: [0, -90, 0, -90, 0, 0]
linear_tcp_speed: 5
linear_tcp_acceleration: 2
joint_speed: 15
joint_acceleration: 7
